src/site/_Site.jsx

```diff
@@ -36,6 +36,14 @@ const propTypes = {
   fetchSearchItems: PropTypes.func,
 };
 
+const getSessionStorage = () => {
+  try {
+    return window.sessionStorage;
+  } catch (e) {
+    return undefined;
+  }
+};
+
```

Comment:
Is this something we can pull out to be re-usable?

Reply:
Moved it to reusable [here](https://github.com/cerner/terra-dev-site/blob/dabf4ceeb0af7bb59540f67283903594e29ac755/src/session.js)